.azure-pipelines/signing.yml

```diff
@@ -7,6 +7,14 @@ parameters:
 
 steps:
 - ${{ if parameters.isWindows }}:
+  - task: PowerShell@2
```

Comment:
Could you please add some variable - to be able to switch it on/off?

Reply:
Added